factcast-store-pgsql/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -29,6 +29,3 @@ databaseChangeLog:
     - include:
          file: factcast/issue346/dropUniqueConstraintForIdent.sql
          relativeToChangelogFile: true
-    - include:
```

Comment:
Is this backwards compatible to an existing 0.1.4.2 installation? Would have assumed the include is one changeset which then would be missing.

(Didn't try yet. Won't be able to test it today. But there's no hurry, right?)

Reply:
no problem there, changes already applied, but missing in changelog are just not taken into account.

Reply:
But please, test this change it in your context before i'll merge. no rush... it has time...